projects/graph/src/graph_demo.py

```diff
@@ -5,12 +5,78 @@
 """
 
 from sys import argv
+from graph import Graph
+from draw import BokehGraph
+import random
 
 
-def main():
-    pass  # TODO
+def createDefaultGraph():
+    graph = Graph()
+    graph.add_vertex('0')
+    graph.add_vertex('1')
+    graph.add_vertex('2')
+    graph.add_vertex('3')
+    graph.add_edge('0', '3')
+    graph.add_edge('0', '1')
+
```

Comment:
add some more verts and edges to this.  I think it might help visualize your bfs and dfs
```suggestion
    graph.add_vertex('4')
    graph.add_vertex('5')
    graph.add_vertex('6')
    graph.add_edge('0', '3')
    graph.add_edge('0', '1')
    graph.add_edge('3', '4')
    graph.add_edge('3', '5')
    graph.add_edge('5', '6')

    graph.bfs('0')
```